selfdrive/test/process_replay/process_replay.py

```diff
@@ -336,12 +338,34 @@ def ublox_rcv_callback(msg):
   ),
 ]
 
+def setup_prefix():
+  os.environ['OPENPILOT_PREFIX'] = str(uuid.uuid4())
+  msg_path = '/dev/shm/' + os.environ['OPENPILOT_PREFIX']
+  try:
+    os.mkdir(msg_path)
+  except FileExistsError:
+    pass
+  except PermissionError:
+    print(f"WARNING: failed to make {msg_path}")
+
+
+def teardown_prefix():
+  params_path = '/data/params/' if os.environ.get('TICI', 0) else os.environ['HOME'] + '/.comma/params/'
+  symlink_path = params_path + os.environ['OPENPILOT_PREFIX']
```

Comment:
this only removes the symlink, not the actual params data

Reply:
https://stackoverflow.com/questions/11700545/how-to-delete-a-symbolic-link-in-python
This post says otherwise.

I can double check